src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4290,6 +4343,11 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
             std::string arg_s = arg;
             ASR::expr_t *value = nullptr;
             ASR::expr_t *init_expr = nullptr;
+            if (i >= default_arg_index_start){
+                size_t default_arg_index = i - default_arg_index_start;
+                this->visit_expr(*(x.m_args.m_defaults[default_arg_index]));
```

Comment:
```suggestion
            AST::expr_t *def_arg = nullptr;
            size_t default_arg_index;
            if (i >= default_arg_index_start){
                default_arg_index = i - default_arg_index_start;
                def_arg = x.m_args.m_defaults[default_arg_index];
                this->visit_expr(*def_arg);
```

We can be a little readable here.